recipes/smhasher/meta.yaml

```diff
@@ -16,6 +16,7 @@ requirements:
     - python
   run:
     - python
+    - libstdcxx-ng # [linux]
```

Comment:
This looks wrong. Can you try to add `- {{ compiler('cxx') }}` to the build dependencies?